chsdi/templates/htmlpopup/vbs_ladestationen.mako

```diff
@@ -46,11 +46,11 @@
     </tr>
     <tr>
         <td class="cell-left">${_('ch.vbs.ladestationen.XKoordinate')}</td>
-        <td>${c['attributes']['x_koordinate'] or '-'}</td>
+        <td>${int(round(c['attributes']['x_koordinate'])) if c['attributes']['x_koordinate'] is not None else '-'}</td>
```

Comment:
Don't these coordinates already come as int values? When I look at the the layer in dev they are always `xyz.0`
In this case I think it makes more sens to integrate them in t the DB as int so you don't have to round them here and use the actual values the data owner provides?

Reply:
Yes it is in the table that need to be changed as well as the model here:
- https://github.com/geoadmin/mf-chsdi3/blob/develop-2024-07-03/chsdi/models/vector/vbs.py#L409
- https://github.com/geoadmin/mf-chsdi3/blob/develop-2024-07-03/chsdi/models/vector/vbs.py#L417-L418

If you add these changes  you will not need this extra code and the make will be easier to read and maintain.

Reply:
yes, the values are integer in the identify result but the htmlpopup converts them for unknown reasons into x.0 numbers...